yarn.lock

```diff
@@ -8219,10 +8224,19 @@ react-scripts@3.1.1:
   optionalDependencies:
     fsevents "2.0.7"
 
+<<<<<<< HEAD
+react-table@^6.10.0:
+  version "6.10.0"
+  resolved "https://registry.yarnpkg.com/react-table/-/react-table-6.10.0.tgz#20444b19d8ca3c1a08e7544e5c3a93e4ba56690e"
+  integrity sha512-s/mQLI1+mNvlae45MfAZyZ04YIT3jUzWJqx34s0tfwpDdgJkpeK6vyzwMUkKFCpGODBxpjBOekYZzcEmk+2FiQ==
+  dependencies:
+    classnames "^2.2.5"
+=======
 react-table@next:
   version "7.0.0-alpha.26"
   resolved "https://registry.yarnpkg.com/react-table/-/react-table-7.0.0-alpha.26.tgz#b131def906c3369d00001b2d7132279785c22942"
   integrity sha512-3Gj+PI0+2SD6K7KSBOj+RzaYETNjPt7zKKxYEBUIMWVvyGbldhtSgNZEGBvzTbQ/dG8d1xXKhyu5/ulh4spfhg==
+>>>>>>> master
```

Comment:
These parts of code:
```
<<<<<<< HEAD
....
=======
....
>>>>>>> master
```
 come from an unresolved merge conflict. 
I can pair program with you to get it resolved properly.